### PR TITLE
Add initial project structure for the `aws.secretmanager` package

### PR DESCRIPTION
**Pull request by ayeshLK**
Part of: [#7471](https://github.com/ballerina-platform/ballerina-library/issues/7471)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,147 @@
...
+
+secretmanager:Client secretmanager = check new(region = secretmanager:US_EAST_1, auth = {
+    accessKeyId,
+    secretAccessKey
+});
+```
+
+### Step 3: Invoke the connector operation
+
+// todo: complete this
```
Comment (Nuvindu) [inline README.md:46]: Do we need to update this?

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
+org.gradle.caching=true
+group=io.ballerina.lib
+version=0.1.0-SNAPSHOT
```
Comment (Nuvindu) [inline gradle.properties:3]: Is this a pre release?

File: gradle/libs.versions.toml
Comment (Nuvindu) [inline gradle/libs.versions.toml:1]: Do we need this file?

File: README.md (hunk 1 above)
Comment (ayeshLK) [reply README.md:46]: Don't hv a content to be update here yet, will do it in a following PR

File: gradle.properties (hunk 2 above)
Comment (ayeshLK) [reply gradle.properties:3]: yeah, this is for a initial version

File: gradle/libs.versions.toml
Comment (ayeshLK) [reply gradle/libs.versions.toml:1]: This was autogenerated by the gradle init task.

**Review by Nuvindu — APPROVED** [review]

**Pull request merged**